src/main/java/xyz/thepathfinder/android/ModelRegistry.java

```diff
@@ -13,6 +14,9 @@
 class ModelRegistry {
 
     private static final Logger logger = Logger.getLogger(ModelRegistry.class.getName());
+    static {
+        logger.setLevel(Level.INFO);
```

Comment:
This sets the minimum level of logs to be displayed? Are you sure you want to hardcode that here? Most libraries expose that to the developer as an option.